<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_listen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>启动服务器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectall">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示/刷新用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectrooms">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示/刷新房间</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="te_show_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>161</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="te_show_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="te_show_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
